docs/book/en/src/dev/rust/networking-https.md

```diff
@@ -0,0 +1,109 @@
+# Networking for Https
+
+The WasmEdge WASI socket API supports HTTP networking in Wasm apps. In order to achieve the goal of supporting HTTPS requests with the same API as HTTP request, we now create a Wasmedge plugin using openssl library. In this chapter, we will give the example of HTTPs requests and explain the design.
```

Comment:
1. the same API as `an` HTTP request.
2. using `the OpenSSL library`.
3. the example of `HTTPS`.

Reply:
OK